### PR TITLE
Suppress unzip warnings

### LINKED ISSUE #910: Set correct file timestamp
Referenced: #845 
----------------------

Uncompressing..tar: ./release.yaml: implausibly old time stamp 1970-01-01 08:00:00
tar: ./bin/heron: implausibly old time stamp 1970-01-01 08:00:00
tar: ./bin: implausibly old time stamp 1970-01-01 08:00:00
tar: ./conf/local/client.yaml: implausibly old time stamp 1970-01-01 08:00:00
tar: ./conf/local/heron_internals.yaml: implausibly old time stamp 1970-01-01 08:00:00
tar: ./conf/local/metrics_sinks.yaml: implausibly old time stamp 1970-01-01 08:00:00
tar: ./conf/local/packing.yaml: implausibly old time stamp 1970-01-01 08:00:00

Though it is a WARNING, it is good if we can remove it.

### PR DESCRIPTION
**Pull request by billonahill**
Fixed #910 but also creates a common_bin.sh` file that we can share logic in across build scripts.

In a follow-up review we can consolidate the duplicate implementations currently copied into each template. For example each file has a method to check for tar, and unzip and some impls are copy/pasted, some are unnecessarily different.

### CODE REVIEW

File: scripts/packages/bin_common.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function set_untar_command {
+  SUPPRESS_TAR_TS_WARNINGS="--warning=no-timestamp"
+  tar $SUPPRESS_TAR_TS_WARNINGS --help &> /dev/null && export TAR_X_FLAGS=$SUPPRESS_TAR_TS_WARNINGS
```
Comment (kramasamy) [inline scripts/packages/bin_common.sh]: why is this help needed?

**Comment by billonahill** [comment]
@kramasamy I added comments in the code explaining why I'm using --help.

**Comment by billonahill** [comment]
Investigating CI failure...

**Comment by kramasamy** [comment]
👍

**Pull request merged**